### ISSUE TITLE
Clicking on error Associated Item doesn't trigger focus

### ISSUE THREAD

**Issue by Cristian-I** [Pretius/Pretius-APEX-Enhanced-Lov-Item#43]
Hey,

If you have a validation on an Enh Lov Item and even if that item is also set as an Associated Item when you click on the error it doesn't focus the item.
![image](https://user-images.githubusercontent.com/14290526/72615953-84fab580-393e-11ea-8826-27bfcd7dcc7d.png)

I think this is more of a nice to have but still it helps a lot the users.

Thank you,
Cristian

**Comment by bostrowski** [comment]
@Cristian-I 
thank you for this feedback. I will look closely at how it can be implemented.

**Comment by michele-pauluzzi** [comment]
it focus on the button for the popup lov

**Comment by bostrowski** [comment]
@Cristian-I 
When popup mode is available it does redirect focus to the popup button (as @michele-pauluzzi already point out). 

I assume your issue describes the situation when ONLY autocomplete mode is available and an item is not focused after clicking on validation message.  Am I right?

This behaviour should be considered as a bug and will be fixed in the nearest release.

**Comment by Cristian-I** [comment]
@bostrowski 
Yes, you are right. This issue happens when only autocomplete mode is selected.

Thank you for looking into this.
